app/scripts/lib/util.ts

```diff
@@ -382,3 +383,37 @@ export function formatValue(
 
   return includeParentheses ? `(${formattedNumber})` : formattedNumber;
 }
+
+type MethodData = {
+  name: string;
+  params: { type: string }[];
+};
+
+export const getMethodDataName = async (
```

Comment:
Could this also go in `shared/lib/four-byte.ts` to keep all the related logic in one place and usable in both the UI and background?

Reply:
This has logic besides getting data from api and this I would prefer to keep it in utils.